src/optimizer/mod.rs

```diff
@@ -42,6 +44,8 @@ impl Optimizer {
         }
         let hep_optimizer = HeuristicOptimizer { rules };
         plan = hep_optimizer.optimize(plan);
+        let out_types_num = plan.out_types().len();
+        plan = plan.prune_col(BitSet::from_iter(0..out_types_num));
```

Comment:
I think we already have `bitvec`, and there's no need to import another bit-vec-like crate...

Reply:
Here we might need a collection which is more like a "Set", Maybe we will change it to [fixedbitset](https://crates.io/crates/fixedbitset)